packages/manager/modules/overthebox/src/overthebox/details/overTheBox-details.html

```diff
@@ -14,21 +14,6 @@
                 ></span>
             </button>
             <oui-dropdown-content>
-                <button
-                    type="button"
-                    class="btn btn-link text-left"
-                    data-ng-click="OverTheBoxDetails.LaunchAction('upgrade')"
-                    data-ng-disabled="!OverTheBoxDetails.availableAction.upgrade"
-                >
-                    <span
-                        class="d-block"
-                        data-translate="overTheBox_action_upgrade_title"
```

Comment:
Both could be also removed from the translations file I guess.

- overTheBox_action_upgrade_title
- overTheBox_action_upgrade_label

Could you please correct me if I'm wrong? Thanks!

Reply:
Yes you're right